### PR TITLE
add support for rgba hex values in color table

### PR DESCRIPTION
**Pull request by thomasguenzel**
For a project I had to adjust the alpha values based on the theme and I think this feature might also be helpful to others.

### CODE REVIEW

File: DKNightVersion/ColorTable/DKColorTable.m [hunk 1]
```diff
@@ -18,7 +18,11 @@ @interface DKColorTable ()
 @implementation DKColorTable
 
 UIColor *DKColorFromRGB(NSUInteger hex) {
-    return [UIColor colorWithRed:((float)((hex & 0xFF0000) >> 16))/255.0 green:((float)((hex & 0xFF00) >> 8))/255.0 blue:((float)(hex & 0xFF))/255.0 alpha:1.0];
+	return [UIColor colorWithRed:((CGFloat)((hex >> 16) & 0xFF)/255.0) green:((CGFloat)((hex >> 8) & 0xFF)/255.0) blue:((CGFloat)(hex & 0xFF)/255.0) alpha:1.0];
```
Comment (Draveness) [inline DKNightVersion/ColorTable/DKColorTable.m]: There are some indent problem in this pull request, have you test this?

Comment (thomasguenzel) [inline DKNightVersion/ColorTable/DKColorTable.m]: @Draveness I've corrected the indentation. I've tested it using red green and blue with and without an alpha channel.

Comment (Draveness) [inline DKNightVersion/ColorTable/DKColorTable.m]: I think the indent should be 4 spaces instead of six, you can see this in this project

Comment (thomasguenzel) [inline DKNightVersion/ColorTable/DKColorTable.m]: @Draveness It is, just look at the newest commit dfadc69

**Comment by Draveness** [comment]
Thx, and this will be added in 2.3.0

**Pull request merged**